### DISCUSSION TITLE
How to Update Torch to 2.1.1?

### DISCUSSION THREAD

**Discussion by gohan2091** [vladmandic/sdnext#2554] [Q&A] [answered]
I would appreciate information on how to update SD.Next to torch 2.1.1? currently I am on 2.1.0+cu121. I ran the command 

">pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121" 

but it said it's already satisfied but SD.next still shows 2.1.0. Any ideas? Also is there now a reason to update python? I am running  3.10.11.

**Comment by vladmandic** [comment, marked as answer]
There is no reason to upgrade either torch or python, but if you really want to, delete venv so new version is installed from scratch.
